docs/_posts/2016-11-17-new-for-4.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "New for 4.0"
+date:   2016-11-17 15:19:00
+---
+
+TSLint [4.0][0] has been released! With this release comes a few exciting [changes][1]. Some of the highlights:
+* **Fixers** - Do you dread turning on a new rule because of all of the new errors? For some of the most common issues, we'll fix them for you. To use this feature, run `tslint` with the `--fix` option. Here are the rules we fix right now:
+    * [array-type][2]
+    * [arrow-parens][3]
+    * [no-unused-variable][4] (for imports)
+    * [no-var-keyword][5]
+    * [ordered-imports][6]
+    * [semicolon][7]
+    * [trailing-comma][8]
+* **Linting `.js` files** - Make migrating from JavaScript even easier! Just add a `jsRules` [section][9] to your `tslint.json` file.
```

Comment:
this isn't exactly the purpose... the purpose here is so that users don't have to use two linters (if they are considering tslint while migrating, they almost certainly were using eslint before). also note that it was a highly requested community feature (we didn't drive it)